support-models/src/main/scala/com/gu/support/workers/states/CreateZuoraSubscriptionState.scala

```diff
@@ -10,11 +11,13 @@ import com.gu.support.encoding.CustomCodecs.{decodeLocalTime, encodeLocalTime}
 case class CreateZuoraSubscriptionState(
   requestId: UUID,
   user: User,
+  giftRecipient: Option[GiftRecipient],
   product: ProductType,
   paymentMethod: PaymentMethod,
   firstDeliveryDate: Option[LocalDate],
   promoCode: Option[PromoCode],
-  salesForceContact: SalesforceContactRecord,
+  salesForceContact: SalesforceContactRecord, //TODO: Remove this it is redundant now we have gifting
```

Comment:
is this hard or not urgent or just not got around to it yet? (I noticed due to fixing up it:test failures)